app/frontend/src/features/messages/groupchats/GroupChatView.tsx

```diff
@@ -78,13 +100,21 @@ export default function GroupChatView() {
   const groupChatMembersQuery = useUsers(groupChat?.memberUserIdsList ?? []);
 
   const {
-    data: messages,
+    data: messagesRes,
     isLoading: isMessagesLoading,
     error: messagesError,
-  } = useQuery<Message.AsObject[], GrpcError>(
+    fetchNextPage,
+    isFetchingNextPage,
+    hasNextPage,
+  } = useInfiniteQuery<GetGroupChatMessagesRes.AsObject, GrpcError>(
```

Comment:
Damn, I can't imagine how much trickier this will be if we were still using Redux. So nice this has barely changed the core of the code 🤩